### PR TITLE
ipcamera_driver: 0.1.1-1 in 'kinetic/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by alireza-hosseini**
Increasing version of package(s) in repository `ipcamera_driver` to `0.1.1-1`:

- upstream repository: https://github.com/alireza-hosseini/ipcamera_driver.git
- release repository: https://github.com/alireza-hosseini/ipcamera_driver-release
- distro file: `kinetic/distribution.yaml`
- bloom version: `0.6.4`
- previous version for package: `null`

## ipcamera_driver

- No changes

### CODE REVIEW

File: kinetic/distribution.yaml [hunk 1]
```diff
@@ -4160,6 +4160,22 @@ repositories:
...
+      type: git
+      url: https://github.com/alireza-hosseini/ipcamera_driver.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/alireza-hosseini/ipcamera_driver-release
+      version: 0.1.1-1
+    source:
+      test_pull_requests: true
```
Comment (mikaelarguedas) [inline kinetic/distribution.yaml]: Just checking: Have you followed the [instructions](http://wiki.ros.org/buildfarm/Pull%20request%20testing) to provide access to this repo or the org unit to the @ros-pull-request-builder user? If not, please either remove this line or provide the necessary access as described in the wiki page.

Comment (alireza-hosseini) [reply kinetic/distribution.yaml]: @mikaelarguedas Yes, I did a few minutes ago.

Comment (mikaelarguedas) [reply kinetic/distribution.yaml]: Awesome thanks for confirming :+1:

File: kinetic/distribution.yaml [hunk 2]
```diff
@@ -4160,6 +4160,22 @@ repositories:
...
     status: developed
+  ipcamera_driver:
+    doc:
+      type: git
+      url: https://github.com/alireza-hosseini/ipcamera_driver.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/alireza-hosseini/ipcamera_driver-release
```
Comment (mikaelarguedas) [inline kinetic/distribution.yaml]: travis is failing because this URL is missing the `.git` extension

**Review by mikaelarguedas — APPROVED** [review]
lgtm

**Pull request merged**